core/include/tangram/platform.h

```diff
@@ -98,12 +98,12 @@ class Platform {
     using UrlRequestId = uint64_t;
 
     // To be called by implementations to pass UrlResponse
-    void onUrlResponse(const UrlRequestHandle _request, UrlResponse&& _response);
+    void onUrlResponse(UrlRequestHandle _request, UrlResponse&& _response);
 
-    virtual void cancelUrlRequestImpl(const UrlRequestId _id) = 0;
+    virtual void cancelUrlRequestImpl(UrlRequestId _id) = 0;
 
     // Return true when UrlRequestId has been set (i.e. when request is async and can be canceled)
-    virtual bool startUrlRequestImpl(const Url& _url, const UrlRequestHandle _request, UrlRequestId& _id) = 0;
+    virtual bool startUrlRequestImpl(const Url& _url, UrlRequestHandle _request, UrlRequestId& _id) = 0;
```

Comment:
I've kind of liked the idea to communicate that if one wants to change `_request` handle one is doing something wrong. Not needed for me though.

Reply:
I see what you mean, but I find that it looks too much like a typo (e.g. you actually meant to make it a `const &`) and `clang-tidy` warns about it.